src/lib/registration/sw_mgmt.rb

```diff
@@ -45,6 +45,7 @@ module Registration
   Yast.import "Installation"
   Yast.import "PackageCallbacks"
   Yast.import "Popup"
+  Yast.import "AutoinstConfig"
```

Comment:
Move the module import to the code below so we do not need it in installed system.